apps/nextjs-website/src/components/ApiViewer/ApiViewer.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC } from 'react';
+import { API } from '@stoplight/elements';
+import '@stoplight/elements/styles.min.css';
+
+type ApiViewerProps = {
+  url: string;
+};
+export const ApiViewer: FC<ApiViewerProps> = ({ url }) => {
+  return (
+    <API
+      apiDescriptionUrl={url}
+      router={typeof window === 'undefined' ? 'memory' : 'history'}
+    />
```

Comment:
```suggestion
    <API
      apiDescriptionUrl={url}
      hideTryIt={true}
      router={typeof window === 'undefined' ? 'memory' : 'history'}
    />
```

Reply:
As described in the User Story, the try it functionality should not be enabled at the moment

Reply:
Here also you should add the `basePath` property (check the [documentation](https://github.com/stoplightio/elements/blob/main/docs/getting-started/elements/elements-options.md) to get more information) because if you land to the API page and then refresh the page, the path in the browser changes.

The `basePath` property should prevent that behaviour.

Reply:
Also, the `export` dropdown menu on the top right of the Elements UI is not present in Figma; we should hide it as well.